dockerfiles/init/manifests/che.pp

```diff
@@ -66,6 +67,7 @@
   $system_super_privileged_mode=getValue("SYSTEM_SUPER__PRIVILEGED__MODE", "false")
 
   $che_keycloak_admin_require_update_password=getValue("CHE_KEYCLOAK_ADMIN_REQUIRE_UPDATE_PASSWORD", "true")
+  $che_keycloak_auth__server__url=getValue("CHE_KEYCLOAK_AUTH__SERVER__URL","")
```

Comment:
I do not see where $che_keycloak_auth__server__url is used, do we need that?

Reply:
yes, i think this is a leftover